soh/soh/Enhancements/debugger/debugSaveEditor.h

```diff
@@ -348,7 +348,7 @@ const std::vector<FlagTable> flagTables = {
         { 0x24, "Market Crowd Text Randomizer" },
         { 0x30, "Entered the Market" },
     } },
-   { "Randomizer Inf Flags", RANDOMIZER_INF, 0x08, {
+   { "Randomizer Inf Flags", RANDOMIZER_INF, 0x09, {
```

Comment:
No because this gets used in this for loop so it should be `ceil(RAND_INF_MAX / 16) - 1`.
I've updated the comment in `randomizer_inf,h` to reflect this
```c
for (int j = 0; j < flagTable.size + 1; j++)
```